pages/testnets/testnet_info.md

```diff
@@ -51,8 +55,8 @@ The above info can be found in this [`v4-testnets` repository](https://github.co
   "feeCurrencies": [
     {
       "coinDenom": "DV4TNT",
-      "coinDecimals": 6,
-      "coinMinimalDenom": "dv4tnt",
+      "coinDecimals": 18,
+      "coinMinimalDenom": "adv4tnt",
```

Comment:
Not 100% what this is used for but from the name, probably should add USDC here?